Support/bone101/UI/bbui.js

```diff
@@ -618,24 +618,24 @@ var UI = (function() {
 
             bar.draw = function(){
                 var len = bars.length;
-                canvas.Bar.ctx.clearRect(0,0,canvas.Bar.e.width,canvas.Bar.e.height);
-                    canvas.Bar.ctx.fillStyle= 'rgb(205,205,205)';
-                    canvas.Bar.ctx.fillRect(bars[len-1].locX,bars[len-1].locY, bars[len-1].length, bars[len-1].height);
-                    canvas.Bar.ctx.fillStyle= bars[len-1].barColor;
-                    canvas.Bar.ctx.fillRect(bars[len-1].locX,bars[len-1].locY, bars[len-1].sliderX() - bars[len-1].locX, 15);
-                    canvas.Bar.ctx.fillStyle= 'rgb(30,30,30)';
-                    canvas.Bar.ctx.fillRect(bars[len-1].sliderX()-2,bars[len-1].locY,14,15);
-                    canvas.Bar.ctx.strokeStyle= bars[len-1].outline;
-                    canvas.Bar.ctx.lineWidth = 2;
-                    canvas.Bar.ctx.strokeRect(bars[len-1].locX,bars[len-1].locY, bars[len-1].length, bars[len-1].height);
-                    canvas.Bar.ctx.fillStyle= 'black';
-                    canvas.Bar.ctx.strokeStyle = 'rgb(225,225,225)';
-                    canvas.Bar.ctx.lineWidth = 6;
-                    canvas.Bar.ctx.font = '12pt Andale Mono';
-                    canvas.Bar.ctx.strokeText(bars[len-1].text, bars[len-1].length + bars[len-1].locX + 5, 
-                        bars[len-1].height + bars[len-1].locY -2);  
-                    canvas.Bar.ctx.fillText(bars[len-1].text, bars[len-1].length + bars[len-1].locX + 5, 
-                        bars[len-1].height + bars[len-1].locY -2);
+                //canvas.Bar.ctx.clearRect(0,0,canvas.Bar.e.width,canvas.Bar.e.height);
+                canvas.Bar.ctx.fillStyle= 'rgb(205,205,205)';
```

Comment:
this is the old code removed now, I made just a tab indentation between function name and it's contents.